introduction.md

```diff
@@ -1,3 +1,8 @@
 this is my intro , i am jamaga
 
-my fav path in the codeschool is Ruby path
\ No newline at end of file
+my fav path in the codeschool is Ruby path and JavaScript, Rails
+
+skills:
+* Ruby
+
+* Rails
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?